projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -790,7 +790,7 @@ on production builds, the attributes are being reordered, causing side-effects
 }
 
 .contact-form__error {
-	margin-bottom: 0.25rem;
+	margin-bottom: var(--wp--style--block-gap, 1.5rem);
```

Comment:
I don't know if we should use `--wp--style--block-gap`; that will not be set for classic, non block-based themes.

Reply:
It's used throughout `grunion.css`. Shoud I update it everywhere?

> that will not be set for classic, non block-based themes.
What'd be the best way to test that?

Reply:
All instances in the stylesheet implement the `1.5rem` fallback, which is what's eventually used in the Twenty Twenty theme.

I think it's safe as is, but I don't mind getting rid of the variable if you think it's safer down the road.

Reply:
Should we worry about other classic themes, that may not have any value defined?

Reply:
I don't think I understand what you mean by that:
> that may not have any value defined?

If I understand correctly, in the worst-case scenario, `var(--wp--style--block-gap, 1.5rem)` will evaluate to `1.5rem`.